README.md

```diff
@@ -291,6 +291,116 @@ do this on [my personal site](https://github.com/kentcdodds/kentcdodds.com)
 which is generated at build-time on the server. Learn about rendering
 [`react` on the server][react-ssr] and [`glamor` too][glamor-ssr].
 
+### Example Style Objects
+
+Style objects can affect pseudo-classes and pseduo-elements, complex CSS
+selectors, introduce keyframe animations, and use media queries:
+
+<details>
+  <summary>pseudo-class</summary>
+
+  ```javascript
+    const MyLink = glamorous.a({
+      ':hover': {
+        color: 'red'
+      }
+    })
+
+    // Use in a render function
+    <MyLink href="https://github.com">GitHub</MyLink>
+  ```
+</details>
+
+<details>
+  <summary>pseudo-element</summary>
+
+  ```javascript
+    const MyListItem = glamorous.li({
+      listStyleType: 'none',
+      position: 'relative',
+      '&::before': {
+        content: `'#'`, // be sure the quotes are included in the passed string
+        display: 'block',
+        position: 'absolute',
+        left: '-20px',
+        width: '20px',
+        height: '20px'
+      }
+    })
+    // Use in a render function
+    <ul>
+      <MyListItem>Item 1</MyListItem>
+      <MyListItem>Item 2</MyListItem>
+      <MyListItem>Item 3</MyListItem>
+    </ul>
+  ```
+</details>
+
+<details>
+  <summary>Relational CSS Selectors</summary>
+
+  ```javascript
+    const MyDiv = glamorous.div({
```

Comment:
Otherwise there's unnecessary indentation in the code block:

<img width="229" alt="screen shot 2017-04-05 at 4 54 57 pm" src="https://cloud.githubusercontent.com/assets/1500684/24730391/a7c7f464-1a20-11e7-85e5-4b8bb952c016.png">